woocommerce-services.php

```diff
@@ -577,6 +582,27 @@ public function jetpack_on_plugins_loaded() {
 		public function on_plugins_loaded() {
 			$this->load_textdomain();
 
+			$is_woo_shipping_active = in_array( 'woocommerce-shipping/woocommerce-shipping.php', get_option( 'active_plugins' ) );
+			$is_woo_tax_active      = in_array( 'woocommerce-tax/woocommerce-tax.php', get_option( 'active_plugins' ) );
+
+			if ( $is_woo_shipping_active || $is_woo_tax_active ) {
+				add_action(
+					'admin_notices',
+					function () use ( $is_woo_shipping_active, $is_woo_tax_active ) {
+						if ( $is_woo_shipping_active && $is_woo_tax_active ) {
+							$active_plugins = __( 'Woo Shipping and Woo Tax plugins are already active.', 'woocommerce-services' );
+						} elseif ( $is_woo_shipping_active ) {
+							$active_plugins = __( 'Woo Shipping plugin is already active.', 'woocommerce-services' );
+						} else {
+							$active_plugins = __( 'Woo Tax plugin is already active.', 'woocommerce-services' );
+						}
+
+						echo '<div class="error"><p><strong>' . sprintf( esc_html__( '%s Please deactivate WooCommerce Shipping & Tax.', 'woocommerce-services' ), esc_html( $active_plugins ) ) . '</strong></p></div>';
```

Comment:
Blocking: AFAIR all `sprintf` should have an a translators comment, so translators knows what they're working with 😄 

```suggestion
						/* translators: %s: the reason WCS&T has to be deactivated */
						echo '<div class="error"><p><strong>' . sprintf( esc_html__( '%s Please deactivate WooCommerce Shipping & Tax.', 'woocommerce-services' ), esc_html( $active_plugins ) ) . '</strong></p></div>';
```